### DISCUSSION TITLE
How save a drawing from vlm?

### DISCUSSION THREAD

**Discussion by harutidmore** [peterdsharpe/AeroSandbox#105] [Q&A] [answered]
How can I save a drawing from VLM.draw()

**Comment by peterdsharpe** [comment, marked as answer]
`asb.VortexLatticeMethod.draw()` will return a PyVista `Plotter` object, which can then be used to save a screenshot:

```python
vlm = asb.VortexLatticeMethod(...)
aero = vlm.run()
plotter = vlm.draw(show=False)

plotter.save_graphic(filename='drawing.svg')
```

See [the PyVista Plotter docs](https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.save_graphic.html) for more options!
